### PR TITLE
*.*: Replace stdout with stderr for human consumable messages.

### PR DESCRIPTION
**Pull request by yifan-gu**

### CODE REVIEW

File: lib/backend/repository/repository.go [hunk 1]
```diff
@@ -302,7 +302,7 @@ func getLayer(imgID, registry string, repoData *RepoData, imgSize int64, tmpDir
 	progressReader := &ioprogress.Reader{
 		Reader:       res.Body,
 		Size:         imgSize,
-		DrawFunc:     ioprogress.DrawTerminalf(os.Stdout, fmtfunc),
+		DrawFunc:     ioprogress.DrawTerminalf(os.Stderr, fmtfunc),
```
Comment (jonboulle) [inline lib/backend/repository/repository.go]: This is a lib and really shouldn't be printing to arbitrary streams - it should probably accept an optional io.Writer that users of the lib can inject. Would you mind filing a followup issue?

**Comment by jonboulle** [comment]
LGTM for now

File: lib/backend/repository/repository.go (hunk 1 above)
Comment (yifan-gu) [inline lib/backend/repository/repository.go]: ? Is providing the stream via DrawFunc not enought?

Comment (jonboulle) [inline lib/backend/repository/repository.go]: What I mean is that users of docker2aci should not have it printing to stderr unless they request it

Comment (yifan-gu) [inline lib/backend/repository/repository.go]: ok. I opened an issue for this and will follow up.

**Comment by jonboulle** [comment]
@yifan-gu thank you!

**Pull request merged**